src/game/client/components/binds.cpp

```diff
@@ -3,19 +3,25 @@
 #include <engine/config.h>
 #include <engine/shared/config.h>
 #include "binds.h"
-
-const int CBinds::s_aDefaultBindKeys[] = { // only simple binds
-	KEY_F1, KEY_F2, KEY_TAB, 'u', KEY_F10,
-	'a', 'd',
-	KEY_SPACE, KEY_MOUSE_1, KEY_MOUSE_2, KEY_LSHIFT, KEY_RSHIFT, KEY_RIGHT, KEY_LEFT,
-	'1', '2', '3', '4', '5',
-	KEY_MOUSE_WHEEL_UP, KEY_MOUSE_WHEEL_DOWN,
-	't', 'y', 'x',
-	KEY_F3, KEY_F4,
-	'r',
+int M[4][5] =
```

Comment:
Can be removed?

Reply:
Oops, that sure is a strange stray array